xml/docu_styleguide.structure.xml

```diff
@@ -1497,23 +1621,40 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
 
 &lt;/glossary&gt;</screen>
     </example>
+    <example xml:id="ex-glossary-adoc" outputformat="adoc">
```

Comment:
Regarding glossaries (not specific to this example): Maybe add some text to link to TermWeb? This may help to keep definitions consistent.

Reply:
@dariavladykina can u pls suggest one or two sentences here?